database/migrations/20200530063307-create-farmers.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.](https://app.codacy.com/gh/BuildForSDG/Farm-Pro/pullRequest?prid=5674718)